README.md

```diff
@@ -181,6 +182,31 @@ Optional steps:
 [bash completions]: extra/completions/liquidctl.bash
 
 
+## Installing on FreeBSD
```

Comment:
It seems that DragonFly BSD [has a port too](https://repology.org/project/python:liquidctl/versions), with the same maintainer.

Can you check that your instructions are (at least partially) applicable to it too and, that being true, adjust this section to cover both?

Reply:
DragonFlyBSD is a separate operating system with its own rules, and I cannot verify its functionality. With that said, I will add a note that it is included in the DragonFly Ports Collection via the FreeBSD Ports Collection and should have matching dependencies. Check for that shortly.

Explanation: Since DragonFlyBSD is a (distant) fork of FreeBSD, its developers are able to run sophisticated scripts to bring the massive FreeBSD Ports Collection to their own OS. That is (almost certainly) why liquidctl is listed as having the same maintainer, even though that person might not even know that his/her work is also being used on DragonFlyBSD (all ports work is done under open BSD licenses).

Reply:
Oh, i didn't know this process was (possibly) automated, and that the two can vary that substantially.

In that case, "Installing on FreeBSD" is fine.